jest-runner-eslint.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  cliOptions: {
+    format: "node_modules/eslint-formatter-pretty"
```

Comment:
In their README, they just do `format=pretty`: https://github.com/sindresorhus/eslint-formatter-pretty#eslint-cli

Would that work here too? I feel bad about hardcoding the path to a `node_module`.

Reply:
Ah didn't know that, let me try

Reply:
Works!